production.Dockerfile

```
@@ -0,0 +1,60 @@
+
+# Switch to the non-privileged user to run the application.
+USER appuser
+
+# Copy the source code into the container.
+COPY . .
+
+# Expose the port that the application listens on.
+EXPOSE 8000
+
+# Run the application.
+CMD gunicorn 'todo_project.wsgi' --bind=0.0.0.0:8000
```

Hadolint (2.12.0)

[warning] Use arguments JSON notation for CMD and ENTRYPOINT arguments

(DL3025)